runtime/src/bank.rs

```diff
@@ -7005,4 +7010,46 @@ mod tests {
         }
         info!("results: {:?}", results);
     }
+
+    #[test]
```

Comment:
@ryoqun how about this test?  I ran it across different scenarios (with the owner in the hash, without, with my archiver's removed w and w/o this PR) and it looks like it accomplishes what we need.  Let me know what you think.

Reply:
perfect! That's what I'm looking for. :)